kernel/packages/unity-interface/UnityInterface.ts

```diff
@@ -60,12 +60,15 @@ function fillMouseEventDataWrapper(eventStruct: any, e: any, target: any) {
 export let targetHeight: number = 1080
 
 function resizeCanvas(module: any) {
+  let desiredHeight = 1080
+
   if (targetHeight > 2000) {
     targetHeight = window.innerHeight * devicePixelRatio
+    desiredHeight = targetHeight
+  } else {
+    desiredHeight = targetHeight - (window.outerHeight - window.innerHeight) * devicePixelRatio
   }
```

Comment:
since `desiredHeight` will be always modified, can we just set one of the possibilities at the beginning to reduce the if/else?

```suggestion
function resizeCanvas(module: any) {
  let desiredHeight = targetHeight - (window.outerHeight - window.innerHeight) * devicePixelRatio

  if (targetHeight > 2000) {
    targetHeight = window.innerHeight * devicePixelRatio
    desiredHeight = targetHeight
  }
```

Reply:
Changed the code but not exactly what you suggested. Didn't want to put the expensive operation as the default one and also had some doubts about the desired/target height as the names weren't exactly friendly, but in any case, the code should be way more readable now.